react-app/src/index.js

```diff
@@ -1,10 +1,32 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import React, { Fragment } from "react";
 
-ReactDOM.render(<App />, document.getElementById('root'));
+import ReactDOM from "react-dom";
+import { BrowserRouter as Router } from "react-router-dom";
+
+import { Provider } from "react-redux";
```

Comment:
same about the imports here, I notice you are grouping them by concern which intuitively makes sense, but keep to what I posted in my previous comment